test/Transactions/Orleans.Transactions.Azure.Test/FaultInjectionConsistencyTests.cs

```diff
@@ -0,0 +1,134 @@
+﻿using Orleans.TestingHost;
+using Orleans.Transactions.Tests;
+using Orleans.Transactions.Tests.Consistency;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Orleans.Transactions.AzureStorage.Tests
+{
+    [TestCategory("BVT"), TestCategory("Transactions")]
```

Comment:
Azure tests should be marked azure, and not be marked bvt unless we're confident that they will pass consistently.

Reply:
ok